package.json

```diff
@@ -41,7 +43,11 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "@wordpress/dependency-extraction-webpack-plugin": "^3.3.1",
     "10up-toolkit": "^2.1.1",
     "cypress-localstorage-commands": "^1.6.1"
+  },
+  "10up-toolkit": {
+    "libraryName": "@10up/block-components"
```

Comment:
This is not needed as you're not creating a UMD bundle. If you were creating a UMD bundle the library name would be used to expose lib via a global var e.g: `window.TenUpBlockComponents`.

UMD bundles are only created if you specify `umd:main` field in package.json.

Reply:
Done 👍